From 97f29f6e74e25916b1cfea20bc2e676409f7f877 Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Sat, 11 Apr 2020 07:58:04 +0530
Subject: [PATCH] Remove some dead code

These doesn't seem to be used anywhere else
The mShowCurrent is initialised as true and mWithBackgroundColor values isn't assigned anywhere.
Change-Id: I096c90e9d3ebe561595adc572e150adde4bba417
---
 .../systemui/omni/DetailedWeatherView.java    | 41 +++----------------
 1 file changed, 6 insertions(+), 35 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/omni/DetailedWeatherView.java b/packages/SystemUI/src/com/android/systemui/omni/DetailedWeatherView.java
index ec7715d1d42..b02dcf60c02 100644
--- a/packages/SystemUI/src/com/android/systemui/omni/DetailedWeatherView.java
+++ b/packages/SystemUI/src/com/android/systemui/omni/DetailedWeatherView.java
@@ -81,8 +81,6 @@ public class DetailedWeatherView extends FrameLayout {
     private TextView mForecastText4;
     private ActivityStarter mActivityStarter;
     private OmniJawsClient mWeatherClient;
-    private boolean mWithBackgroundColor;
-    private boolean mShowCurrent = true;
     private View mCurrentView;
     private TextView mCurrentTemp;
     private TextView mCurrentText;
@@ -93,12 +91,6 @@ public class DetailedWeatherView extends FrameLayout {
     private View mWeatherLine;
     private TextView mProviderName;
 
-    /** The background colors of the app, it changes thru out the day to mimic the sky. **/
-    public static final String[] BACKGROUND_SPECTRUM = { "#212121", "#27232e", "#2d253a",
-            "#332847", "#382a53", "#3e2c5f", "#442e6c", "#393a7a", "#2e4687", "#235395", "#185fa2",
-            "#0d6baf", "#0277bd", "#0d6cb1", "#1861a6", "#23569b", "#2d4a8f", "#383f84", "#433478",
-            "#3d3169", "#382e5b", "#322b4d", "#2c273e", "#272430" };
-
     public DetailedWeatherView(Context context) {
         this(context, null);
     }
@@ -151,10 +143,6 @@ public class DetailedWeatherView extends FrameLayout {
         mWeatherLine = findViewById(R.id.current_weather);
         mProviderName = (TextView) findViewById(R.id.current_weather_provider);
 
-        if (!mShowCurrent) {
-            mCurrentView.setVisibility(View.GONE);
-        }
-
         mEmptyViewImage.setOnLongClickListener(new View.OnLongClickListener() {
             @Override
             public boolean onLongClick(View v) {
@@ -202,14 +190,7 @@ public class DetailedWeatherView extends FrameLayout {
         String format = DateFormat.is24HourFormat(mContext) ? "HH:mm" : "hh:mm a";
         SimpleDateFormat sdf = new SimpleDateFormat(format);
         mWeatherTimestamp.setText(getResources().getString(R.string.omnijaws_service_last_update) + " " + sdf.format(timeStamp));
-        if (mShowCurrent) {
-            mWeatherData.setText(weatherData.windSpeed + " " + weatherData.windUnits + " " + weatherData.pinWheel +" ⋅ " +
-                    weatherData.humidity);
-        } else {
-            mWeatherData.setText(weatherData.temp + weatherData.tempUnits + " ⋅ " +
-                    weatherData.windSpeed + " " + weatherData.windUnits + " " + weatherData.pinWheel +" ⋅ " +
-                    weatherData.humidity);
-        }
+        mWeatherData.setText(weatherData.windSpeed + " " + weatherData.windUnits + " " + weatherData.pinWheel +" ⋅ " + weatherData.humidity);
 
         sdf = new SimpleDateFormat("EE");
         Calendar cal = Calendar.getInstance();
@@ -257,17 +238,12 @@ public class DetailedWeatherView extends FrameLayout {
         mForecastTemp4.setText(formatTempString(weatherData.forecasts.get(4).low, weatherData.forecasts.get(4).high, weatherData.tempUnits));
         mForecastText4.setText(dayShort);
 
-        if (mShowCurrent) {
-            d = mWeatherClient.getWeatherConditionImage(weatherData.conditionCode);
-            d = overlay(mContext.getResources(), d);
-            mCurrentImage.setImageDrawable(d);
-            mCurrentTemp.setText(formatTempString(weatherData.temp, null, weatherData.tempUnits));
-            mCurrentText.setText(mContext.getResources().getText(R.string.omnijaws_current_text));
-        }
+        d = mWeatherClient.getWeatherConditionImage(weatherData.conditionCode);
+        d = overlay(mContext.getResources(), d);
+        mCurrentImage.setImageDrawable(d);
+        mCurrentTemp.setText(formatTempString(weatherData.temp, null, weatherData.tempUnits));
+        mCurrentText.setText(mContext.getResources().getText(R.string.omnijaws_current_text));
 
-        if (mWithBackgroundColor) {
-            setBackgroundColor(getCurrentHourColor());
-        }
     }
 
     private Drawable overlay(Resources resources, Drawable image) {
@@ -305,11 +281,6 @@ public class DetailedWeatherView extends FrameLayout {
         mWeatherClient.updateWeather();
     }
 
-    public static int getCurrentHourColor() {
-        final int hourOfDay = Calendar.getInstance().get(Calendar.HOUR_OF_DAY);
-        return Color.parseColor(BACKGROUND_SPECTRUM[hourOfDay]);
-    }
-
     private void setErrorView() {
         mEmptyView.setVisibility(View.VISIBLE);
         mWeatherLine.setVisibility(View.GONE);
